<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Szerokość</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSpinBox" name="widthInput">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Wysokość</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QSpinBox" name="heightInput">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="drawButton">
        <property name="text">
         <string>Rysuj</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="simulateButton">
        <property name="text">
         <string>Rozpocznij symulację</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QComboBox" name="startingCompositionSelect">
        <item>
         <property name="text">
          <string comment="0">Ręczny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="1">Niezmienny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="2">Glider</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="3">Oscylator</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="4">Losowy</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTableView" name="celluralTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
